src/tools/tests/goldens/WriterReaderExample.kt

```diff
@@ -17,9 +17,7 @@ object IngestionPlan : Plan(
             "arcs.core.data.testdata.Reader",
             mapOf(
                 "data" to HandleConnection(
-                    StorageKeyParser.parse(
-                        "reference-mode://{db://25e71af4e9fc8b6958fc46a8f4b7cdf6b5f31516@arcs/Thing}{db://25e71af4e9fc8b6958fc46a8f4b7cdf6b5f31516@arcs/!:writingArcId/handle/my-handle-id}"
-                    ),
+                    CreateableStorageKey("my-handle-id", Capabilities.Persistent),
```

Comment:
@mmandlis Does this look right?

Reply:
no, IngestionPlan is a long running arc (aka singleton recipe).
its storage keys need to be generated at compile time.

this applies to all changes in this file.

Reply:
Thanks for checking, I've found / implemented a fix.